btcpay-server/umbrel-app.yml

```diff
@@ -34,11 +34,10 @@ path: ""
 defaultUsername: ""
 defaultPassword: ""
 releaseNotes: >
-  This update from version 1.7.12 to 1.9.2 includes a range of new features, including customizable checkout forms and invoice metadata, store branding options,
-  a redesigned Point of Sale keypad view, improved Point of Sale receipt, and more!
-
+  This update from version 1.9.2 to 1.11.0 includes a range of new features, including an overhaul of invoice reporting, Point of Sale cart redesign, 
+  adding product categories to Point of Sale apps, playing a sound when an invoice is paid, and more!
 
   A full list of new features, improvements, and bug fixes
-  for versions between 1.7.12 and 1.9.2 can be found here: https://github.com/btcpayserver/btcpayserver/releases.
+  for versions between 1.9.2 and 1.11.0 can be found here: https://github.com/btcpayserver/btcpayserver/releases.
```

Comment:
Nice release notes!

Can you please make two small changes:
1. version can now be 1.11.1
2. add an extra newline between the paragraphs. 2 new lines will mean that in the UI, the paragraphs will be separated by a single newline (finicky, I know). Here's what it looks like when only 1 newline is set in the .yml:

<img width="1292" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/b7ecebf6-6f3a-4869-8c88-fed0beb7a992">

Reply:
Done!